.eslintrc.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
   reportUnusedDisableDirectives: true,
   ignorePatterns: ['examples'],
   parser: '@typescript-eslint/parser',
-  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended', 'standard', 'prettier'],
+  extends: ['eslint:recommended', 'standard', 'plugin:@typescript-eslint/recommended', 'prettier'],
```

Comment:
`eslint-config-standart` is a config for javascript files (for typescript we must use this https://github.com/standard/eslint-config-standard-with-typescript), due `standard` was placed after `plugin:@typescript-eslint/recommended` there are some false-positive errors because `@typescript-eslint/recommended` turn-out some rules and standard turn-in